students/JeffBennett/Lesson03/mailroom.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+# Data
+
+donors = [
+    ['Lorenzo Currie', 3000, 250, 5000],
+    ['Marie Kane', 1000, 195, 50.50],
+    ['John Beresford Tipton', 10000, 3000, 2500],
+    ['Janna Adams', 35.50, 19.75],
+    ['LeSean Haskell', 100],
+    ['Laila Samir', 1000, 250]
+]
+
+# Input/Output
+
+
+def print_menu():
+    """print menu of user choices."""
+    print('\nMain Menu\n' + '-'*25 +
+          '\n(1)\tSend a Thank You\n(2)\tCreate a Report\n(3)\tQuit')
+
+
+def print_donor_list(donors):
+    """print list of donor names."""
+    print('Our Current List of Donors:\n' + '-' * 25)
+    for item in donors:
+        print(item[0])
+
+
+def print_email(donor, gift):
+    """print email thanking donor for recent donation."""
+    print(f'\nDear {donor}\n')
+    print('We are delighted to acknowledge your generous')
+    print(f'gift of ${gift:,.2f} to Northwest Lifeboats.')
+    print('Your gift supports fishermen all along the Washington coast.\n')
+    print('Sincerely yours\n\nJ. A. Bennett\nDirector')
```

Comment:
You could use triple-quoted string to preserve white space across multiple lines